docs/release/release-tasks.md

```diff
@@ -310,30 +310,38 @@ The goal of this task to make the book for the current release available under e
    <br>Prior art: [Add release 1.2 book link](https://github.com/kubernetes-sigs/cluster-api/pull/6697)
 
 #### Create PR for release notes
-
+1. Checkout the `main` branch.
+1. Build the release note tools binary.
+   ```bash
+   go build -o bin/notes hack/tools/release/notes.go
+   ```
 1. Checkout the latest commit on the release branch, e.g. `release-1.4`, or the main branch if the release branch doesn't yet exist (e.g. beta release).
-2. Generate release notes with:
+1. Generate release notes with:
 
    ```bash
    # PREVIOUS_TAG should be the last patch release of the previous minor release.
    PREVIOUS_TAG=v1.3.x
    # RELEASE_TAG should be the new desired tag (note: at this point the tag does not yet exist).
-   RELEASE_TAG=v1.4.x make release-notes
+   RELEASE_TAG=v1.4.x
+   # If this is a beta or RC release, add the --pre-release-version flag
```

Comment:
do you need to set `--pre-release-version` flag, i.e `--pre-release-version=v1.6.0-beta.0`?

Reply:
`--pre-release-version` is just a boolean flag
It enables/disables the "pre release" block text, but it doesn't take any value as argument